Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.py

```diff
@@ -0,0 +1,340 @@
+                readable_output=human_readable,
+                raw_response=results_response
+            )
+            command_results.append(command_result)
+    return command_results
+
+
+def command_update_alert(client: Client, args: dict) -> CommandResults:
+    """Update the status or comment of an alert.
+
+    Args:
+        client (Client): Client object with request
+        args (dict): Usually demisto.args()
+
+    Returns:
+        CommandResults.
+    """
+    status = str(args.get('status'))
+    comment = str(args.get('comment'))
+    uuid_val = str(args.get('uuid'))
+    if not uuid_val:
+        raise DemistoException("Please provide 'uuid' parameter.")
+    if status == 'None' and comment == 'None':
+        raise DemistoException("Please provide either 'status' or 'comment' parameter.")
+    columns = {}
+    if status != 'None':
+        columns['status'] = status
+    if comment != 'None':
+        columns['comment'] = comment
+    data = {
+        "table_name": "alert",
+        "columns": columns,
+        "primary_key_columns": ["uuid_"],
+        "primary_key_values": [[uuid_val]]
+    }
+    response = client.update_alert(data)
+    return CommandResults(
+        outputs_prefix='ThreatstreamAlerts.UpdateAlert',
+        outputs=response,
+        readable_output=tableToMarkdown(name="Update Alert", t=response, removeNull=True),
+        raw_response=response
+    )
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+    Perform basic request to check if the connection to service was successful.
+    Raises:
+        exceptions if something goes wrong.
+
+    Args:
+        Client: client to use
+
+    Returns:
+        'ok' if the response is ok, else will raise an error
+    """
+    try:
+        client.check_connection()
+        return "ok"
+    except Exception as e:
+        raise DemistoException(f"Error in API call - check the username and the API Key. Error: {e}.")
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    """main function, parses params and runs command functions"""
+
+    params = demisto.params()
+    base_url = params.get("url")
+    verify_certificate = not argToBoolean(params.get("insecure", False))
+    proxy = argToBoolean(params.get("proxy", False))
+
+    command = demisto.command()
+
+    try:
+        username = params.get("credentials", {}).get("identifier")
+        api_key = params.get("credentials", {}).get("password")
+        client = Client(
+            server_url=base_url,
+            username=username,
+            api_key=api_key,
+            verify=verify_certificate,
+            proxy=proxy
+        )
+        args = demisto.args()
+        commands = {
+            'anomali-security-analytics-search-job-create': command_create_search_job,
+            'anomali-security-analytics-search-job-results': command_get_search_job_results,
+            'anomali-security-analytics-update-alert': command_update_alert,
```

Comment:
Command names should follow conventions - product-object-operator 
```suggestion
            'anomali-security-analytics-alert-update': command_update_alert,
```
Change it in the yml as well.